### ISSUE TITLE
m4a conversion does not work

### ISSUE THREAD

**Issue by rsr-at-mindtwin** [openaudible/openaudible#356]
decided to try m4a, noting that it immediately fails:

```
May 09, 2020 2:25:29 PM org.openaudible.desktop.swt.manager.menu.CommandCenter execute
INFO: Command: Convert
May 09, 2020 2:25:29 PM org.openaudible.util.queues.ThreadedQueue run
INFO: Starting job convert masked In progress:1 queue: 0 threads:
May 09, 2020 2:25:29 PM org.openaudible.Audible jobStarted
INFO: ConvertQueue started:masked size:0
May 09, 2020 2:25:29 PM org.openaudible.util.SimpleProcess <init>
INFO: Running: /Applications/OpenAudible.app/Contents/Resources/app/bin/mac/ffmpeg -i "/Users/foobie/archive/openaudible/aax/masked.AAX" 
May 09, 2020 2:25:29 PM org.openaudible.convert.ConvertJob doConversion
INFO: converting: masked to temp file: /Users/foobie/Library/OpenAudible/tmp/BK_MUMBLE_NUM_temp.m4a from source /Users/foobie/archive/openaudible/aax/masked.AAX length=228491847
May 09, 2020 2:25:29 PM org.openaudible.convert.ConvertJob doConversion
INFO: /Applications/OpenAudible.app/Contents/Resources/app/bin/mac/ffmpeg -activation_bytes BYTES -i "/Users/foobie/archive/openaudible/aax/masked.AAX" -c:a copy -c:v copy -map_metadata 0 /Users/foobie/Library/OpenAudible/tmp/BK_MUMBLE_NUM_temp.m4a 
May 09, 2020 2:25:29 PM org.openaudible.convert.ConvertJob doConversion
INFO: create M4A /Users/foobie/archive/openaudible/aax/masked.AAX result:1
May 09, 2020 2:25:29 PM org.openaudible.convert.ConvertJob doConversion
SEVERE: conversion got non-zero result:1 
stdErr:er, ...    Stream #0:2: Video: mjpeg, yuvj420p(pc, bt470bg/unknown/unknown), 500x500 [SAR 1:1 DAR 1:1], 90k tbr, 90k tbn, 90k tbc, [ipod @ 0x7f7fe0800000] Could not find tag for codec mjpeg in stream #0, codec not currently supported in container, Could not write header for output file #0 (incorrect codec parameters ?): Invalid argument, Stream mapping:,   Stream #0:2 -> #0:0 (copy),   Stream #0:0 -> #0:1 (copy),     Last message repeated 1 times
May 09, 2020 2:25:29 PM org.openaudible.convert.ConvertJob convert
SEVERE: Error converting book to MP3:masked
java.io.IOException: Failed to convert /Users/foobie/archive/openaudible/aax/masked.AAX. Check console for details.
  at org.openaudible.convert.ConvertJob.doConversion(ConvertJob.java:317)
  at org.openaudible.convert.ConvertJob.convert(ConvertJob.java:379)
  at org.openaudible.convert.ConvertJob.processJob(ConvertJob.java:432)
  at org.openaudible.util.queues.ThreadedQueue$JobThread.run(ThreadedQueue.java:420)

May 09, 2020 2:25:29 PM org.openaudible.util.queues.ThreadedQueue jobError
INFO: ConvertQueue error processing convert masked
java.io.IOException: Failed to convert /Users/foobie/archive/openaudible/aax/masked.AAX. Check console for details.
  at org.openaudible.convert.ConvertJob.doConversion(ConvertJob.java:317)
  at org.openaudible.convert.ConvertJob.convert(ConvertJob.java:379)
  at org.openaudible.convert.ConvertJob.processJob(ConvertJob.java:432)
  at org.openaudible.util.queues.ThreadedQueue$JobThread.run(ThreadedQueue.java:420)

May 09, 2020 2:25:29 PM org.openaudible.Audible jobCompleted
INFO: ConvertQueue completed:masked remaining queue size:0
May 09, 2020 2:25:29 PM org.openaudible.desktop.swt.manager.AudibleGUI checkAutomation
INFO: Start auto conversion:12
May 09, 2020 2:25:29 PM org.openaudible.desktop.swt.manager.AudibleGUI getTaskString
WARNING: error converting masked: Failed to convert /Users/foobie/archive/openaudible/aax/masked.AAX. Check console for details.

```

the command line output, when run by hand:

```
ffmpeg version N-89585-g5533cbf-tessus Copyright (c) 2000-2017 the FFmpeg developers
  built with Apple LLVM version 8.0.0 (clang-800.0.42.1)
  configuration: --cc=/usr/bin/clang --prefix=/opt/ffmpeg --extra-version=tessus --enable-avisynth --enable-fontconfig --enable-gpl --enable-libass --enable-libbluray --enable-libfreetype --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopus --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzmq --enable-libzvbi --enable-version3 --pkg-config-flags=--static --disable-ffplay
  libavutil      56.  6.100 / 56.  6.100
  libavcodec     58.  8.100 / 58.  8.100
  libavformat    58.  3.100 / 58.  3.100
  libavdevice    58.  0.100 / 58.  0.100
  libavfilter     7.  7.100 /  7.  7.100
  libswscale      5.  0.101 /  5.  0.101
  libswresample   3.  0.101 /  3.  0.101
  libpostproc    55.  0.100 / 55.  0.100
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x7ff1a2800000] [aax] file checksum == removed [but was valid]
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x7ff1a2800000] stream 0, timescale not set
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/Users/foobie/archive/openaudible/aax/masked.AAX':
  Metadata:
    major_brand     : aax 
    minor_version   : 1
    compatible_brands: aax M4B mp42isom
    creation_time   : 2020-03-04T06:23:13.000000Z
    comment         : Chapter 8
    genre           : Audiobook
    date            : 2020
  Duration: 07:27:37.63, start: 0.000000, bitrate: 128 kb/s
    Chapter #0:0: start 0.000000, end 14.512472
    Metadata:
      title           : Chapter 1
    Chapter #0:1: start 14.512472, end 4080.024671
    Metadata:
      title           : Chapter 2
    Chapter #0:2: start 4080.024671, end 8387.047619
    Metadata:
      title           : Chapter 3
    Chapter #0:3: start 8387.047619, end 13075.040363
    Metadata:
      title           : Chapter 4
    Chapter #0:4: start 13075.040363, end 17140.297143
    Metadata:
      title           : Chapter 5
    Chapter #0:5: start 17140.297143, end 20973.818776
    Metadata:
      title           : Chapter 6
    Chapter #0:6: start 20973.818776, end 26829.101859
    Metadata:
      title           : Chapter 7
    Chapter #0:7: start 26829.101859, end 26857.628345
    Metadata:
      title           : Chapter 8
    Stream #0:0(eng): Audio: aac (LC) (aavd / 0x64766161), 44100 Hz, stereo, fltp, 125 kb/s (default)
    Metadata:
      creation_time   : 2020-03-04T06:23:13.000000Z
      handler_name    : ?Apple Sound Media Handler
    Stream #0:1(eng): Data: bin_data (text / 0x74786574) (default)
    Metadata:
      creation_time   : 2020-03-04T06:23:14.000000Z
      handler_name    : ?Apple Text Media Handler
    Stream #0:2: Video: mjpeg, yuvj420p(pc, bt470bg/unknown/unknown), 500x500 [SAR 1:1 DAR 1:1], 90k tbr, 90k tbn, 90k tbc
[ipod @ 0x7ff1a180e400] Could not find tag for codec mjpeg in stream #0, codec not currently supported in container
Could not write header for output file #0 (incorrect codec parameters ?): Invalid argument
Stream mapping:
  Stream #0:2 -> #0:0 (copy)
  Stream #0:0 -> #0:1 (copy)
    Last message repeated 1 times

```

(I have masked the identifying data for the file, but it was valid)

**Comment by openaudible** [comment]
I am unable to reproduce.. but it I think that might be a cover art problem. Can you try a pre-release?
https://openaudible.org/prerelease

**Comment by simudvarac** [comment]
I have exactly the same bug and exactly the same console output.
Tried the prerelaeas with almost same result:

SCHWERWIEGEND: Error converting book to MP3:Das kleine Kaninchen, das so ger
java.io.IOException: Failed to convert /Users/mac/Library/OpenAudible/aax/Das kleine Kaninchen, das so gerne einschlafen möchte Die ideale Einschlafhilfe für Ihr Kind.1.AAX. Check console for details.
	at org.openaudible.convert.ConvertJob.doConversion(ConvertJob.java:317)
	at org.openaudible.convert.ConvertJob.convert(ConvertJob.java:379)
	at org.openaudible.convert.ConvertJob.processJob(ConvertJob.java:432)
	at org.openaudible.util.queues.ThreadedQueue$JobThread.run(ThreadedQueue.java:420)

Mai 10, 2020 12:10:44 NACHM. org.openaudible.util.queues.ThreadedQueue jobError
INFORMATION: ConvertQueue error processing convert Das kleine Kaninchen, das so ger
java.io.IOException: Failed to convert /Users/mac/Library/OpenAudible/aax/Das kleine Kaninchen, das so gerne einschlafen möchte Die ideale Einschlafhilfe für Ihr Kind.1.AAX. Check console for details.
	at org.openaudible.convert.ConvertJob.doConversion(ConvertJob.java:317)
	at org.openaudible.convert.ConvertJob.convert(ConvertJob.java:379)
	at org.openaudible.convert.ConvertJob.processJob(ConvertJob.java:432)
	at org.openaudible.util.queues.ThreadedQueue$JobThread.run(ThreadedQueue.java:420)

Mai 10, 2020 12:10:44 NACHM. org.openaudible.Audible jobCompleted
INFORMATION: ConvertQueue completed:Das kleine Kaninchen, das so ger remaining queue size:0
Mai 10, 2020 12:10:44 NACHM. org.openaudible.desktop.swt.manager.AudibleGUI getTaskString
WARNUNG: error converting Das kleine Kaninchen, das so ger: Failed to convert /Users/mac/Library/OpenAudible/aax/Das kleine Kaninchen, das so gerne einschlafen möchte Die ideale Einschlafhilfe für Ihr Kind.1.AAX. Check console for details.

**Comment by Geordon** [comment]
I'm having similar issues.  I have uploaded what I think is the whole console output for a particuular file here: https://gist.github.com/Geordon/41288ee5db49bea1894e42e80f639ed6

**Comment by rsr-at-mindtwin** [comment]
pre-release has the same problem:

```
May 10, 2020 10:31:11 AM org.openaudible.desktop.swt.gui.GUI <init>
INFO: Starting OpenAudible build 2.0.6 for cocoa swt 4928 jvm 12 file.encoding UTF-8
May 10, 2020 10:31:14 AM org.openaudible.desktop.swt.manager.AudibleGUI <init>
INFO: audible desktop 2.0.6
May 10, 2020 10:31:14 AM org.openaudible.Directories getPrefs
INFO: loaded settings 1550346683529
May 10, 2020 10:31:15 AM org.openaudible.Audible load
INFO: Loading /Users/foobie/Dropbox/foobie/openaudible/books.json
May 10, 2020 10:31:16 AM org.openaudible.Directories getPrefs
INFO: loaded settings 1550346683529
May 10, 2020 10:31:16 AM org.openaudible.Audible checkLibrary
INFO: loaded 735 books.
May 10, 2020 10:31:16 AM org.openaudible.desktop.swt.manager.AudibleGUI applicationStarted
INFO: applicationStarted took 1502 ms. to complete.
May 10, 2020 10:31:22 AM org.openaudible.desktop.swt.manager.menu.CommandCenter execute
INFO: Command: Preferences
May 10, 2020 10:31:25 AM org.openaudible.desktop.swt.manager.views.Preferences okPressed
INFO: okPressed
May 10, 2020 10:31:25 AM org.openaudible.util.HTMLUtil writeFile
INFO: wrote 336 to disk:/Users/foobie/Library/OpenAudible/settings.json
May 10, 2020 10:31:26 AM org.openaudible.desktop.swt.manager.VersionCheck getVersion
INFO: versionCheck: https://openaudible.org/swt_version.json
May 10, 2020 10:31:45 AM org.openaudible.desktop.swt.manager.menu.CommandCenter execute
INFO: Command: Convert
May 10, 2020 10:31:45 AM org.openaudible.util.queues.ThreadedQueue run
INFO: Starting job convert TheBookTitle In progress:1 queue: 0 threads:
May 10, 2020 10:31:45 AM org.openaudible.Audible jobStarted
INFO: ConvertQueue started:TheBookTitle size:0
May 10, 2020 10:31:45 AM org.openaudible.util.SimpleProcess <init>
INFO: Running: /Applications/OpenAudible.app/Contents/Resources/app/bin/mac/ffmpeg -i "/Users/foobie/Dropbox/foobie/openaudible/aax/TheBookTitle.AAX" 
May 10, 2020 10:31:47 AM org.openaudible.convert.ConvertJob doConversion
INFO: converting: TheBookTitle to temp file: /Users/foobie/Library/OpenAudible/tmp/BK_MUMBLE_temp.m4a from source /Users/foobie/Dropbox/foobie/openaudible/aax/TheBookTitle.AAX length=619484594
May 10, 2020 10:31:47 AM org.openaudible.convert.ConvertJob doConversion
INFO: /Applications/OpenAudible.app/Contents/Resources/app/bin/mac/ffmpeg -activation_bytes BYTES -i "/Users/foobie/Dropbox/foobie/openaudible/aax/TheBookTitle.AAX" -c:a copy -c:v copy -map_metadata 0 /Users/foobie/Library/OpenAudible/tmp/BK_MUMBLE_temp.m4a 
May 10, 2020 10:31:48 AM org.openaudible.convert.ConvertJob doConversion
INFO: create M4A /Users/foobie/Dropbox/foobie/openaudible/aax/TheBookTitle.AAX result:1
May 10, 2020 10:31:48 AM org.openaudible.convert.ConvertJob doConversion
SEVERE: conversion got non-zero result:1 
stdErr::0 -> #0:1 (copy), ...    Last message repeated 1 times
May 10, 2020 10:31:48 AM org.openaudible.convert.ConvertJob convert
SEVERE: Error converting book to MP3:TheBookTitle
java.io.IOException: Failed to convert /Users/foobie/Dropbox/foobie/openaudible/aax/TheBookTitle.AAX. Check console for details.
	at org.openaudible.convert.ConvertJob.doConversion(ConvertJob.java:317)
	at org.openaudible.convert.ConvertJob.convert(ConvertJob.java:379)
	at org.openaudible.convert.ConvertJob.processJob(ConvertJob.java:432)
	at org.openaudible.util.queues.ThreadedQueue$JobThread.run(ThreadedQueue.java:420)

May 10, 2020 10:31:48 AM org.openaudible.util.queues.ThreadedQueue jobError
INFO: ConvertQueue error processing convert TheBookTitle
java.io.IOException: Failed to convert /Users/foobie/Dropbox/foobie/openaudible/aax/TheBookTitle.AAX. Check console for details.
	at org.openaudible.convert.ConvertJob.doConversion(ConvertJob.java:317)
	at org.openaudible.convert.ConvertJob.convert(ConvertJob.java:379)
	at org.openaudible.convert.ConvertJob.processJob(ConvertJob.java:432)
	at org.openaudible.util.queues.ThreadedQueue$JobThread.run(ThreadedQueue.java:420)

May 10, 2020 10:31:48 AM org.openaudible.Audible jobCompleted
INFO: ConvertQueue completed:TheBookTitle remaining queue size:0
May 10, 2020 10:31:49 AM org.openaudible.desktop.swt.manager.AudibleGUI getTaskString
WARNING: error converting TheBookTitle: Failed to convert /Users/foobie/Dropbox/foobie/openaudible/aax/TheBookTitle.AAX. Check console for details.
May 10, 2020 10:31:49 AM org.openaudible.desktop.swt.manager.AudibleGUI getTaskString
WARNING: error converting TheBookTitle: Failed to convert /Users/foobie/Dropbox/foobie/openaudible/aax/TheBookTitle.AAX. Check console for details.
May 10, 2020 10:31:56 AM org.openaudible.desktop.swt.manager.menu.CommandCenter execute
INFO: Command: Console
```

**Comment by rsr-at-mindtwin** [comment]
here's the ffmpeg command like:

```
$ /Applications/OpenAudible.app/Contents/Resources/app/bin/mac/ffmpeg -activation_bytes BYTES -i "/Users/foobie/Dropbox/foobie/openaudible/aax/BookNiceTitle.AAX" -c:a copy -c:v copy -map_metadata 0 ~/Downloads/Video/BK_MUMBLE_NUM_temp.m4a 
ffmpeg version N-89585-g5533cbf-tessus Copyright (c) 2000-2017 the FFmpeg developers
  built with Apple LLVM version 8.0.0 (clang-800.0.42.1)
  configuration: --cc=/usr/bin/clang --prefix=/opt/ffmpeg --extra-version=tessus --enable-avisynth --enable-fontconfig --enable-gpl --enable-libass --enable-libbluray --enable-libfreetype --enable-libgsm --enable-libmodplug --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopus --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libvidstab --enable-libvo-amrwbenc --enable-libvorbis --enable-libvpx --enable-libwavpack --enable-libx264 --enable-libx265 --enable-libxavs --enable-libxvid --enable-libzmq --enable-libzvbi --enable-version3 --pkg-config-flags=--static --disable-ffplay
  libavutil      56.  6.100 / 56.  6.100
  libavcodec     58.  8.100 / 58.  8.100
  libavformat    58.  3.100 / 58.  3.100
  libavdevice    58.  0.100 / 58.  0.100
  libavfilter     7.  7.100 /  7.  7.100
  libswscale      5.  0.101 /  5.  0.101
  libswresample   3.  0.101 /  3.  0.101
  libpostproc    55.  0.100 / 55.  0.100
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x7fcfe8000000] [aax] file checksum == 2d25f12bf805499ca16661b3e49e06aad1b40790
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x7fcfe8000000] stream 0, timescale not set
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from '/Users/foobie/Dropbox/foobie/openaudible/aax/BookNiceTitle.AAX':
  Metadata:
    major_brand     : aax 
    minor_version   : 1
    compatible_brands: aax M4B mp42isom
    creation_time   : 2020-05-09T22:09:33.000000Z
    genre           : Audiobook
    title           : BookNiceTitle
    artist          : authors
    album_artist    : authors
    album           : BookNiceTitle
    comment         : comment
    copyright       : copyright
    date            : 2011
  Duration: 21:16:52.01, start: 0.000000, bitrate: 64 kb/s
    Chapter #0:0: start 0.000000, end 289.645714
    Metadata:
      title           : Chapter 1
    Chapter #0:1: start 289.645714, end 1040.764807
    Metadata:
      title           : Chapter 2
    Chapter #0:2: start 1040.764807, end 1462.996463
    Metadata:
      title           : Chapter 3
    Chapter #0:3: start 1462.996463, end 1560.473832
    Metadata:
      title           : Chapter 4
    Chapter #0:4: start 1560.473832, end 2283.868299
    Metadata:
      title           : Chapter 5
    Chapter #0:5: start 2283.868299, end 3279.447075
    Metadata:
      title           : Chapter 6
    Chapter #0:6: start 3279.447075, end 3422.481995
    Metadata:
      title           : Chapter 7
    Chapter #0:7: start 3422.481995, end 3772.546032
    Metadata:
      title           : Chapter 8
    Chapter #0:8: start 3772.546032, end 4073.708844
    Metadata:
      title           : Chapter 9
    Chapter #0:9: start 4073.708844, end 4418.153651
    Metadata:
      title           : Chapter 10
    Chapter #0:10: start 4418.153651, end 4764.641814
    Metadata:
      title           : Chapter 11
    Chapter #0:11: start 4764.641814, end 5994.091973
    Metadata:
      title           : Chapter 12
    Chapter #0:12: start 5994.091973, end 6308.815238
    Metadata:
      title           : Chapter 13
    Chapter #0:13: start 6308.815238, end 6702.625669
    Metadata:
      title           : Chapter 14
    Chapter #0:14: start 6702.625669, end 8085.095329
    Metadata:
      title           : Chapter 15
    Chapter #0:15: start 8085.095329, end 9032.608798
    Metadata:
      title           : Chapter 16
    Chapter #0:16: start 9032.608798, end 9579.299410
    Metadata:
      title           : Chapter 17
    Chapter #0:17: start 9579.299410, end 9720.987574
    Metadata:
      title           : Chapter 18
    Chapter #0:18: start 9720.987574, end 10747.773968
    Metadata:
      title           : Chapter 19
    Chapter #0:19: start 10747.773968, end 10977.001361
    Metadata:
      title           : Chapter 20
    Chapter #0:20: start 10977.001361, end 11485.936327
    Metadata:
      title           : Chapter 21
    Chapter #0:21: start 11485.936327, end 11943.183673
    Metadata:
      title           : Chapter 22
    Chapter #0:22: start 11943.183673, end 13179.832018
    Metadata:
      title           : Chapter 23
    Chapter #0:23: start 13179.832018, end 13295.420952
    Metadata:
      title           : Chapter 24
    Chapter #0:24: start 13295.420952, end 14129.481723
    Metadata:
      title           : Chapter 25
    Chapter #0:25: start 14129.481723, end 14575.769252
    Metadata:
      title           : Chapter 26
    Chapter #0:26: start 14575.769252, end 14949.656961
    Metadata:
      title           : Chapter 27
    Chapter #0:27: start 14949.656961, end 15219.287075
    Metadata:
      title           : Chapter 28
    Chapter #0:28: start 15219.287075, end 16646.664127
    Metadata:
      title           : Chapter 29
    Chapter #0:29: start 16646.664127, end 17931.470658
    Metadata:
      title           : Chapter 30
    Chapter #0:30: start 17931.470658, end 18109.521270
    Metadata:
      title           : Chapter 31
    Chapter #0:31: start 18109.521270, end 18756.661406
    Metadata:
      title           : Chapter 32
    Chapter #0:32: start 18756.661406, end 18918.643810
    Metadata:
      title           : Chapter 33
    Chapter #0:33: start 18918.643810, end 19726.140952
    Metadata:
      title           : Chapter 34
    Chapter #0:34: start 19726.140952, end 20119.161905
    Metadata:
      title           : Chapter 35
    Chapter #0:35: start 20119.161905, end 20348.435737
    Metadata:
      title           : Chapter 36
    Chapter #0:36: start 20348.435737, end 21007.278730
    Metadata:
      title           : Chapter 37
    Chapter #0:37: start 21007.278730, end 21376.243810
    Metadata:
      title           : Chapter 38
    Chapter #0:38: start 21376.243810, end 21816.494150
    Metadata:
      title           : Chapter 39
    Chapter #0:39: start 21816.494150, end 22557.628662
    Metadata:
      title           : Chapter 40
    Chapter #0:40: start 22557.628662, end 22858.745034
    Metadata:
      title           : Chapter 41
    Chapter #0:41: start 22858.745034, end 23825.577506
    Metadata:
      title           : Chapter 42
    Chapter #0:42: start 23825.577506, end 24496.123356
    Metadata:
      title           : Chapter 43
    Chapter #0:43: start 24496.123356, end 26224.941859
    Metadata:
      title           : Chapter 44
    Chapter #0:44: start 26224.941859, end 27761.777778
    Metadata:
      title           : Chapter 45
    Chapter #0:45: start 27761.777778, end 28770.220408
    Metadata:
      title           : Chapter 46
    Chapter #0:46: start 28770.220408, end 28914.323447
    Metadata:
      title           : Chapter 47
    Chapter #0:47: start 28914.323447, end 29994.655057
    Metadata:
      title           : Chapter 48
    Chapter #0:48: start 29994.655057, end 30631.253333
    Metadata:
      title           : Chapter 49
    Chapter #0:49: start 30631.253333, end 31527.450703
    Metadata:
      title           : Chapter 50
    Chapter #0:50: start 31527.450703, end 32114.683356
    Metadata:
      title           : Chapter 51
    Chapter #0:51: start 32114.683356, end 33332.291338
    Metadata:
      title           : Chapter 52
    Chapter #0:52: start 33332.291338, end 33475.558458
    Metadata:
      title           : Chapter 53
    Chapter #0:53: start 33475.558458, end 34152.559456
    Metadata:
      title           : Chapter 54
    Chapter #0:54: start 34152.559456, end 34241.584762
    Metadata:
      title           : Chapter 55
    Chapter #0:55: start 34241.584762, end 35465.183492
    Metadata:
      title           : Chapter 56
    Chapter #0:56: start 35465.183492, end 36246.813605
    Metadata:
      title           : Chapter 57
    Chapter #0:57: start 36246.813605, end 36387.247891
    Metadata:
      title           : Chapter 58
    Chapter #0:58: start 36387.247891, end 37265.519456
    Metadata:
      title           : Chapter 59
    Chapter #0:59: start 37265.519456, end 37899.702857
    Metadata:
      title           : Chapter 60
    Chapter #0:60: start 37899.702857, end 38269.875374
    Metadata:
      title           : Chapter 61
    Chapter #0:61: start 38269.875374, end 39203.735510
    Metadata:
      title           : Chapter 62
    Chapter #0:62: start 39203.735510, end 39737.144308
    Metadata:
      title           : Chapter 63
    Chapter #0:63: start 39737.144308, end 40938.219683
    Metadata:
      title           : Chapter 64
    Chapter #0:64: start 40938.219683, end 41508.223129
    Metadata:
      title           : Chapter 65
    Chapter #0:65: start 41508.223129, end 42643.400272
    Metadata:
      title           : Chapter 66
    Chapter #0:66: start 42643.400272, end 43530.866939
    Metadata:
      title           : Chapter 67
    Chapter #0:67: start 43530.866939, end 44564.758639
    Metadata:
      title           : Chapter 68
    Chapter #0:68: start 44564.758639, end 44796.075828
    Metadata:
      title           : Chapter 69
    Chapter #0:69: start 44796.075828, end 46178.359728
    Metadata:
      title           : Chapter 70
    Chapter #0:70: start 46178.359728, end 46239.335329
    Metadata:
      title           : Chapter 71
    Chapter #0:71: start 46239.335329, end 47601.975147
    Metadata:
      title           : Chapter 72
    Chapter #0:72: start 47601.975147, end 48127.442721
    Metadata:
      title           : Chapter 73
    Chapter #0:73: start 48127.442721, end 48260.400181
    Metadata:
      title           : Chapter 74
    Chapter #0:74: start 48260.400181, end 49755.718821
    Metadata:
      title           : Chapter 75
    Chapter #0:75: start 49755.718821, end 50187.470658
    Metadata:
      title           : Chapter 76
    Chapter #0:76: start 50187.470658, end 51015.354921
    Metadata:
      title           : Chapter 77
    Chapter #0:77: start 51015.354921, end 51310.016145
    Metadata:
      title           : Chapter 78
    Chapter #0:78: start 51310.016145, end 53167.891156
    Metadata:
      title           : Chapter 79
    Chapter #0:79: start 53167.891156, end 53315.895147
    Metadata:
      title           : Chapter 80
    Chapter #0:80: start 53315.895147, end 54656.197370
    Metadata:
      title           : Chapter 81
    Chapter #0:81: start 54656.197370, end 54897.174059
    Metadata:
      title           : Chapter 82
    Chapter #0:82: start 54897.174059, end 56763.176054
    Metadata:
      title           : Chapter 83
    Chapter #0:83: start 56763.176054, end 57226.460590
    Metadata:
      title           : Chapter 84
    Chapter #0:84: start 57226.460590, end 58181.775964
    Metadata:
      title           : Chapter 85
    Chapter #0:85: start 58181.775964, end 58779.178957
    Metadata:
      title           : Chapter 86
    Chapter #0:86: start 58779.178957, end 59016.068934
    Metadata:
      title           : Chapter 87
    Chapter #0:87: start 59016.068934, end 59807.869388
    Metadata:
      title           : Chapter 88
    Chapter #0:88: start 59807.869388, end 60018.660136
    Metadata:
      title           : Chapter 89
    Chapter #0:89: start 60018.660136, end 60349.219410
    Metadata:
      title           : Chapter 90
    Chapter #0:90: start 60349.219410, end 61321.160272
    Metadata:
      title           : Chapter 91
    Chapter #0:91: start 61321.160272, end 61437.260045
    Metadata:
      title           : Chapter 92
    Chapter #0:92: start 61437.260045, end 61639.366531
    Metadata:
      title           : Chapter 93
    Chapter #0:93: start 61639.366531, end 62033.223401
    Metadata:
      title           : Chapter 94
    Chapter #0:94: start 62033.223401, end 63238.896327
    Metadata:
      title           : Chapter 95
    Chapter #0:95: start 63238.896327, end 63272.611701
    Metadata:
      title           : Chapter 96
    Chapter #0:96: start 63272.611701, end 64110.619864
    Metadata:
      title           : Chapter 97
    Chapter #0:97: start 64110.619864, end 64399.476100
    Metadata:
      title           : Chapter 98
    Chapter #0:98: start 64399.476100, end 65556.758639
    Metadata:
      title           : Chapter 99
    Chapter #0:99: start 65556.758639, end 65676.712925
    Metadata:
      title           : Chapter 100
    Chapter #0:100: start 65676.712925, end 65718.741043
    Metadata:
      title           : Chapter 101
    Chapter #0:101: start 65718.741043, end 66688.963628
    Metadata:
      title           : Chapter 102
    Chapter #0:102: start 66688.963628, end 66957.386304
    Metadata:
      title           : Chapter 103
    Chapter #0:103: start 66957.386304, end 67612.885624
    Metadata:
      title           : Chapter 104
    Chapter #0:104: start 67612.885624, end 67931.834921
    Metadata:
      title           : Chapter 105
    Chapter #0:105: start 67931.834921, end 67988.770249
    Metadata:
      title           : Chapter 106
    Chapter #0:106: start 67988.770249, end 69047.089342
    Metadata:
      title           : Chapter 107
    Chapter #0:107: start 69047.089342, end 69233.684898
    Metadata:
      title           : Chapter 108
    Chapter #0:108: start 69233.684898, end 69888.441179
    Metadata:
      title           : Chapter 109
    Chapter #0:109: start 69888.441179, end 70491.184762
    Metadata:
      title           : Chapter 110
    Chapter #0:110: start 70491.184762, end 70771.496054
    Metadata:
      title           : Chapter 111
    Chapter #0:111: start 70771.496054, end 71392.444082
    Metadata:
      title           : Chapter 112
    Chapter #0:112: start 71392.444082, end 72109.708481
    Metadata:
      title           : Chapter 113
    Chapter #0:113: start 72109.708481, end 72529.757460
    Metadata:
      title           : Chapter 114
    Chapter #0:114: start 72529.757460, end 73808.712562
    Metadata:
      title           : Chapter 115
    Chapter #0:115: start 73808.712562, end 74420.511927
    Metadata:
      title           : Chapter 116
    Chapter #0:116: start 74420.511927, end 76361.514376
    Metadata:
      title           : Chapter 117
    Chapter #0:117: start 76361.514376, end 76505.710295
    Metadata:
      title           : Chapter 118
    Chapter #0:118: start 76505.710295, end 76535.524717
    Metadata:
      title           : Chapter 119
    Chapter #0:119: start 76535.524717, end 76560.137868
    Metadata:
      title           : Chapter 120
    Chapter #0:120: start 76560.137868, end 76612.004989
    Metadata:
      title           : Chapter 121
    Stream #0:0(eng): Audio: aac (LC) (aavd / 0x64766161), 22050 Hz, stereo, fltp, 62 kb/s (default)
    Metadata:
      creation_time   : 2020-05-09T22:06:02.000000Z
      handler_name    : ?Apple Sound Media Handler
    Stream #0:1(eng): Data: bin_data (text / 0x74786574) (default)
    Metadata:
      creation_time   : 2020-05-09T22:06:03.000000Z
      handler_name    : ?Apple Text Media Handler
    Stream #0:2: Video: mjpeg, yuvj420p(pc, bt470bg/unknown/unknown), 500x500 [SAR 1:1 DAR 1:1], 90k tbr, 90k tbn, 90k tbc
File '/Users/foobie/Downloads/Video/BK_MUMBLE_NUM_temp.m4a' already exists. Overwrite ? [y/N] y
[ipod @ 0x7fcfe7803600] Could not find tag for codec mjpeg in stream #0, codec not currently supported in container
Could not write header for output file #0 (incorrect codec parameters ?): Invalid argument
Stream mapping:
  Stream #0:2 -> #0:0 (copy)
  Stream #0:0 -> #0:1 (copy)
    Last message repeated 1 times

```

**Comment by rsr-at-mindtwin** [comment]
FYI, if I use the ffmpeg installed by homebrew and relatively up to date, it works fine (and fast!).

Is it possible you are not testing with the ffmpeg that you are packaging?

homebrew version:

```
ffmpeg version 4.2.2 Copyright (c) 2000-2019 the FFmpeg developers
  built with Apple clang version 11.0.0 (clang-1100.0.33.17)
  configuration: --prefix=/usr/local/Cellar/ffmpeg/4.2.2_2 --enable-shared --enable-pthreads --enable-version3 --enable-avresample --cc=clang --host-cflags= --host-ldflags= --enable-ffplay --enable-gnutls --enable-gpl --enable-libaom --enable-libbluray --enable-libmp3lame --enable-libopus --enable-librubberband --enable-libsnappy --enable-libtesseract --enable-libtheora --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-lzma --enable-libfontconfig --enable-libfreetype --enable-frei0r --enable-libass --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-librtmp --enable-libspeex --enable-libsoxr --enable-videotoolbox --disable-libjack --disable-indev=jack
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100

```

**Comment by openaudible** [comment]
Been doing all my recent testing on Windows. Reproduced on Mac. Getting latest 4.2.1 binaries from ffbinaries.com.  Will test and update. Thank you for the help! Will update ASAP..

**Comment by openaudible** [comment]
Fixed in 2.0.7.  Replaced ffmpeg with ffmpeg 4.2.2. We were using a prerelease to fix a bug in the mp3 lame code. Hopefully all will be fine with this release.
